google/cloud/forseti/scanner/scanners/config_validator_scanner.py

```diff
@@ -191,3 +200,41 @@ def run(self):
                 iam_policy=True):
             # Write iam violations to the db.
             self._output_results(flattened_violations)
+
+    @staticmethod
+    def verify_policy_library():
```

Comment:
This will check that the Policy Library directory exists. It will check that the constraints directories exist and are not empty. If not, then the exception is raised and the scanner is not run.